pcdsdevices/analog_signals.py

```diff
@@ -50,45 +51,53 @@ class Acromag(Device):
     ai1_14 = Cpt(EpicsSignalRO, ":ai1:14", kind='normal')
     ai1_15 = Cpt(EpicsSignalRO, ":ai1:15", kind='normal')
 
-    def initialize_mesh(self, sp_ch, rb_ch, scale=1000.0):
-        """
-        Setup the Acromag for mesh power supply usage
 
-        Parameters
-        ----------
-        sp_ch : int
-            Setpoint Acromag channel to which high voltage supply setpoint
-            is connected. Range is 0 to 15
+class Mesh(Device):
+    """
+    Class for Mesh High Voltage Supply that is connected to
+    Acromag inputs and outputs
 
-        rb_ch : int
-            Read back Acromag channel to which high voltage readback is
-            connected. Range is 0 to 15
+    Parameters
+    ----------
+    prefix : str
+        Prefix of Acromag to be used
 
-        scale : float, optional
-            Gain for high voltage supply to be controlled by the Acromag
-        """
+    sp_ch : int
+        Setpoint Acromag channel to which high voltage supply setpoint
+        is connected. Range is 0 to 15
+
+    rb_ch : int
+        Read back Acromag channel to which high voltage readback is
+        connected. Range is 0 to 15
+
+    scale : float, optional
+        Gain for high voltage supply to be controlled by the Acromag
+
+    """
+
+    write_cpt = FCpt(EpicsSignal, '{self.prefix}' + ':ao1:' + '{self.sp_ch}')
+    read_cpt = FCpt(EpicsSignalRO, '{self.prefix}' + ':ai1:' + '{self.rb_ch}')
```

Comment:
Super super small nitpick, but note that these become signals after instantiation, so calling them `cpt` here instead of `sig` or `ch` seems inaccurate

Reply:
That makes sense to me, just made that change.